frontend/theme/theme.jsx

```diff
@@ -10,2055 +12,2286 @@ import IndeterminateCheckBo from '@/assets/svg/_IndeterminateCheckBoxFilled.svg'
 import InfoOutlined from '@/assets/svg/_InfoOutlined.svg';
 import WarningAmberOutlined from '@/assets/svg/_WarningAmberOutlined.svg';
 
-const globalTheme = createTheme({
-  palette: {
-    50: 'rgba(236, 239, 241, 1)',
-    100: 'rgba(207, 216, 220, 1)',
-    200: 'rgba(176, 190, 197, 1)',
-    300: 'rgba(144, 164, 174, 1)',
-    400: 'rgba(120, 144, 156, 1)',
-    500: 'rgba(96, 125, 139, 1)',
-    600: 'rgba(84, 110, 122, 1)',
-    700: 'rgba(69, 90, 100, 1)',
-    800: 'rgba(55, 71, 79, 1)',
-    900: 'rgba(38, 50, 56, 1)',
-    mode: 'dark',
-    primary: {
-      main: 'rgba(105, 73, 255, 1)',
-      contrastText: 'rgba(255, 255, 255, 0.87)',
-    },
-    secondary: {
-      main: 'rgba(255, 193, 7, 1)',
-      contrastText: 'rgba(255, 255, 255, 0.87)',
-    },
-    tertiary: {
-      main: 'rgba(99,	53,	248, 0.87)',
-      contrastText: 'rgba(99,	53,	248, 0.87)',
-      contrast: 'rgba(99,	53,	248, 0.87)',
-      Shades: {
-        '8p': 'rgba(99,	53,	248, 0.08)',
-        '12p': 'rgba(	99,	53,	248, 0.12)',
-        '16p': 'rgba(	99,	53,	248, 0.16)',
-        '30p': 'rgba(	99,	53,	248, 0.3)',
-        '50p': 'rgba(	99,	53,	248, 0.5)',
-      },
-    },
-    error: {
-      main: 'rgba(244, 67, 54, 1)',
-      contrastText: 'rgba(255, 255, 255, 1)',
-    },
-    warning: {
-      main: 'rgba(255, 167, 38, 1)',
-      contrastText: 'rgba(0, 0, 0, 0.87)',
-    },
-    success: {
-      main: 'rgba(18, 209, 142, 1)',
-      contrastText: 'rgba(0, 0, 0, 0.87)',
-    },
-    info: {
-      main: 'rgba(41, 182, 246, 1)',
-      contrastText: 'rgba(0, 0, 0, 0.87)',
-    },
-    text: {
-      primary: 'rgba(255, 255, 255, 1)',
-      secondary: 'rgba(255, 255, 255, 0.7)',
-      disabled: 'rgba(255, 255, 255, 0.5)',
-      light: 'rgba(0, 0, 0, 1)',
+const defaultPalette = {
```

Comment:
I moved all of the globalTheme palette settings into a defaultPalette object (this is the default dark settings, which contain the text and background configurations)I did this so that I could create separate palettes that could be toggled between based on state